doc/source/structures/collections/list.rst

```diff
@@ -41,6 +41,9 @@ Structure
           - Type
           - Description
 
+        * - All suffixes of :struct:`Enumerable`
+          -
+          - :struct:`List` objects are a type of :struct:`Enumerable`
```

Comment:
Okay I can see that.  It's just that when people do `PRINT LIST(1,2,3):INHERITANCE.` they'll see the Collection type in the printout that's not documented, between List and Enumerable.  Maybe that's not such a big deal?